### ISSUE TITLE
The AWS Toolkit plugin for Rider can't find the aws lambda handler

### ISSUE THREAD

**Issue by yemanasyan** [aws/aws-toolkit-jetbrains#2835]
The AWS Toolkit can't find the handler for my dotnet core 3.1 project. Please check the screenshot with my project's structure:
<img width="2115" alt="Screen Shot 2021-09-28 at 2 03 25 PM" src="https://user-images.githubusercontent.com/4953775/135149849-9767ab8e-67cb-44ee-964d-2f69c991a068.png">

I created an AWS Lambda run configuration like in the following screenshot:
<img width="1542" alt="Screen Shot 2021-09-28 at 2 05 58 PM" src="https://user-images.githubusercontent.com/4953775/135150094-06cea249-3d68-4dd1-ad38-8e843254636c.png">

The plugin cannot find handler 'ReportRenderer::ReportRenderer.Function::ReportRendererHandler' in the project. 
But when I choose the AWS Lambda run configuration and push the debug or run buttons it opens a page similar to run configuration and after setting the handler it works:
<img width="894" alt="Screen Shot 2021-09-28 at 2 09 24 PM" src="https://user-images.githubusercontent.com/4953775/135150475-1ed540a4-edb2-4c68-b22e-55f7c2055703.png">

Even if it works it doesn't save the handler and any time I have to specify it.
How I can fix it on my side? Will you be able to ship a fix for this issue?

**Your Environment**
- OS: OSX Big Sur
- JetBrains product: Rider
- JetBrains product version: 2021.2.1
- AWS Toolkit version: 1.32-212
- SAM CLI version: 1.32.0
- JVM/Python version: openjdk 11.0.11

**Comment by yemanasyan** [comment]
@abrooksv do I have enough info here in this ticket or something is missing?

**Comment by abrooksv** [comment]
Yes, I believe there is everything I need to try to reproduce it

**Comment by tdankert** [comment]
Same issue here with Rider 2021.3.4
OS: Manjaro Linux
AWS Toolkit: 1.41-213
SAM CLI: 1.42.0

**Comment by fabiobeoni** [comment]
Same here with:

- Ubuntu: 22.04.3 LTS
- WebStorm: Build #WS-221.6008.13, built on July 19, 2022
- AWS Toolkit: 1.67-221
- Labda Runtime: NodeJS 18

Sample project, just to try out the AWS Toolkit.. Lambda file is in the root of the project as you can see below:

![lambda](https://github.com/aws/aws-toolkit-jetbrains/assets/16882089/f4bf449a-8bc4-453b-bfad-d5b05b71bb1a)

**Comment by fabiobeoni** [comment]
Hi guys... no progress on this issue since a long time... Can you please explain what's the expected string input for the "Handler" input field? Thanks.

**Comment by fabiobeoni** [comment]
Good to know, many thanks.
